macros/filter_bots.sql

```diff
@@ -10,17 +10,31 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
 {%- endmacro -%}
 
 {% macro default__filter_bots(table_alias = none) %}
+  {% if var('snowplow__enable_iab', false) %}
+    and (not iab__spider_or_robot = True 
```

Comment:
How about something like
`and coalesce(iab__spider_or_robot, False ) = False`

Reply:
Sure, I like this option, cleaner as well as easier on the evaluation side due to removing or.